### DISCUSSION TITLE
The education certification has been rejected 7 times and still has not been approved

### DISCUSSION THREAD

**Discussion by yuuu-d** [community/community#50979] [GitHub Education]
I applied for education certification and provided **my student ID card photo**, Chinese and English versions of **the academic transcript verification report provided by the Ministry of Education,** and an **electronic transcript with the school seal**. However, none of them passed the certification. And I applied using my educational email address.
What should I do in this situation? I have no idea how to pass this certification. 
I followed the prompt to change my certification documents, and modified my GitHub user profile by changing the profile picture and name to my own (although I have since changed it back and do not want to use my own picture as my profile picture). However, I still failed to pass the certification. 
Can you please advise me on what I can do to pass the certification?
I have been rejected **7 times** and have tried all the solutions I can think of.

**Comment by samus-aran** [comment]
Hi @ZhaoHanYuuu 👋 

Happy you're here lets see how we can assist ⭐ 

What was the last rejection reasons you received and we can start from there 👍

**Comment by whitewatercn** [comment]
可以参考我的这个帖子再逝一逝 https://xlog.whitewater.wang/Github_Student_Developer_Pack
You can refer to this post to try again https://xlog.whitewater.wang/Github_Student_Developer_Pack
